pkg/api/controller.go

```diff
@@ -5176,6 +5176,26 @@ func (c *Controller) PostStatsEvents(w http.ResponseWriter, r *http.Request, bod
 	writeResponse(w, r, http.StatusNoContent, nil)
 }
 
+func (c *Controller) ListPullRequests(w http.ResponseWriter, r *http.Request, repository string, params apigen.ListPullRequestsParams) {
+	writeResponse(w, r, http.StatusNotImplemented, nil)
+}
+
+func (c *Controller) CreatePullRequest(w http.ResponseWriter, r *http.Request, body apigen.CreatePullRequestJSONRequestBody, repository string) {
+	writeResponse(w, r, http.StatusNotImplemented, nil)
+}
+
+func (c *Controller) DeletePullRequest(w http.ResponseWriter, r *http.Request, repository string, pullRequest string) {
```

Comment:
Should be `repositoryID` and `pullRequestID`.
Also relevant for `GetPullRequest` and `UpdatePullRequest`.

Reply:
For repository our convention in all other functions is `repository`
I'll update `pullRequest`